cloud-data-protection/src/components/onboarding/onboarding.tsx

```diff
@@ -79,14 +155,60 @@ const OnboardingComponent = () => {
                 </div>
             }
             {onboarding?.status === OnboardingStatus.AccountConnected &&
-                <p>
-                Congratulations! Your Google Workspace account is connected. Please choose a backup scheme that fits your business.
-                </p>
+                <div className='onboarding__scheme'>
+                    Congratulations! Your Google Workspace account is connected. Please choose a backup scheme that fits your business.
+                    {schemes?.length &&
+                        <form className='onboarding__scheme__form' onSubmit={(e) => handleOnboardingSchemeSubmit(e)}>
+                            <RadioGroup className='onboarding__scheme__select' onChange={(e) => setSchemeId(parseInt(e.target.value))} value={schemeId}>
+                                {schemes.map(scheme =>
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7398694)
```suggestion
                                {schemes.map((scheme) =>
```